packages/wallets/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCard.scss

```
@@ -99,3 +99,14 @@
         }
     }
 }
+
+.wallets-dynamic-leverage-modal-transition {
+    &-enter {
+        transform: rotateY(-180deg);
+        transition: transform 0.6s ease;
+    }
+
+    &-enter-active {
```

If its an effect or modifier, we should use `--`:
```suggestion
    &-enter--active {
```

---

The CSSTransition library uses this suffix for animation, I don't think we can change them

---

I have changed the implementation to use CSS animate. Not using the CSSTransition Library anymore.